### DISCUSSION TITLE
List names should be longer than 32 chars or count emoji names as 1 char

### DISCUSSION THREAD

**Discussion by obvionaoe** [community/community#9904] [Other Feature Feedback, Questions, & Ideas]
The title is self explanatory.

I tried to create a list called `:globe_with_meridians: Extensions` and this renders as ":globe_with_meridians: Extensions" which should fit as a list name.

Thanks, loving this feature so far

**Comment by pirate** [comment]
I agree, I think List names should allowed to be as long as Issue names, or at least longer than they are now.

<img width="527" alt="Screenshot 2024-02-05 at 10 35 39 PM" src="https://github.com/community/community/assets/511499/cc533fc4-c683-4d9b-8bea-43da357adb47">

**Comment by Phuchong0259** [comment]
ไม่เอาก็ได้ครับ ผมทำไม่เป็น  ผมจบแค่ ป.6 คอมพิวเตอร์ก็ไม่มีมีโทรศัพท์ที่เแาไว้เทรด
